gen/container_test.go

```diff
@@ -1306,14 +1312,16 @@ func TestContainerValidate(t *testing.T) {
 	}
 
 	for _, tt := range tests {
-		value, err := tt.value.ToWire()
-		if err == nil {
-			err = wire.EvaluateValue(value) // lazy error
-		}
+		t.Run(tt.desc, func(t *testing.T) {
+			value, err := tt.value.ToWire()
+			if err == nil && value.String() != "" {
```

Comment:
`value.String` check unnecessary. The implementation of `wire.Value.String()` either returns a non-empty string or panics.

```suggestion
			if err == nil {
```